internal/keycloakb/mfa_tools.go

```diff
@@ -9,8 +9,8 @@ import (
 
 var notMFATypes = map[string]bool{"password": true, "password-history": true}
 
-// CheckRemovableMFA checks if a given credential is removable (owned by user and not the last MFA)
-func CheckRemovableMFA(ctx context.Context, credentialID string, getCredentials func() ([]kc.CredentialRepresentation, error), logger Logger) error {
+// CheckRemovableMFA checks if a given credential is removable (owned by user and not the password credential)
```

Comment:
The comment should be adapted as we do not check if the credential is owned by user